test/functional/mempool_accept.py

```diff
@@ -327,13 +306,13 @@ def run_test(self):
         self.log.info('A transaction that is locked by BIP68 sequence logic')
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_reference)))
         tx.vin[0].nSequence = 2  # We could include it in the second block mined from now, but not the very next one
+        tx.nVersion = 2
```

Comment:
Why not add version to `send_self_transfer()` as optional argument too?

Reply:
I thought about that approach, but this is the only time I needed to modify `nVersion` and I didn't know how viable that would be considering other files that didn't modify `nVersion`.  Nevertheless, if it's overall consensus is to include it in `send_self_transfer()`, I can happily do it!

Reply:
I did try to do that, but I wasn't sure about the stylistic aspect of passing arguments in functions. Do we have a hard cap on the # of arguments we can pass on functions?

Reply:
Maybe it's worth just assuming `nVersion = 2`, and add it as an option argument in the future if/when there's a test that requires `nVersion = 1`?